### PR TITLE
Feature/Projects category filter

### PR DESCRIPTION
**Pull request by juanbravozu**
## Changes

- Projects are filtered depending on which category is selected on route.
- Category link for all projects now highlights when there's no category selected.
- Added styles to display a message when no project is displayed.
- Added constant for _All projects_ category.
- Added option to getPageData function to retrieve other kinds of content.

image
image

### CODE REVIEW

File: components/ProjectWrapper.js [hunk 1]
```diff
@@ -24,7 +25,11 @@ export default function ProjectWrapper({
             >
               <CategoryLink
                 Component="a"
-                highlight={link === currCategory}
+                highlight={
+                  link === currCategory
+                  || (currCategory === undefined
```
Comment (Cruzlex7) [inline components/ProjectWrapper.js:30]: could you try with ` !currCategory`

File: utils/constants.js [hunk 2]
```diff
@@ -8,3 +8,4 @@ export const ALLIES_SECTION = 'alliesHomeSection';
 export const CONTACT_LABEL = 'Contáctanos';
 export const HERO_SECTION = 'hero';
 export const BASE_URL = 'https://cdn.contentful.com/spaces';
+export const ALL_PROJECTS = 'todos-los-proyectos';
```
Comment (Cruzlex7) [inline utils/constants.js:11]: add context to the constant, what about 'ALL_PROJECTS_CATEGORY_FILTER'

**Review by javiermlovo — APPROVED** [review]

**Review by javiermlovo — APPROVED** [review]

**Review by Cruzlex7 — APPROVED** [review]

File: components/Projects.js [hunk 3]
```diff
@@ -1,20 +1,31 @@
...
+            variation={index % 2 !== 0
+              ? ['reverse', 'project']
+              : 'project'}
+          />
+        ))}
+
+        {projectsList.length === 0
+          && (
+          <div className="project project--empty">
+            Aún no hay proyectos en esta categoría
```
Comment (tatarodriguez11) [inline components/Projects.js:24]: éste texto podrias sacarlo en una constante local, no donde tienes las demás constantes

Comment (juanbravozu) [reply components/Projects.js]: Done

File: components/Projects.js [hunk 4]
```diff
@@ -1,20 +1,33 @@
 import Project from './Project';
 
-export default function Projects({ projectsList: { project: list } }) {
+const NO_PROJECTS_MESSAGE = 'Aún no hay proyectos en esta categoría';
```
Comment (tatarodriguez11) [inline components/Projects.js:3]: porfa agregalo dentro del scope de Projects

Comment (juanbravozu) [reply components/Projects.js]: Done

**Review by tatarodriguez11 — APPROVED** [review]

**Pull request merged**